tests/index.js

```diff
@@ -52,19 +49,22 @@ describe('<ContentLoader />:', () => {
       })
     })
 
-    describe('when type is custom', () => {
+  })
+
+  describe('when type is custom', () => {
+
       const wrapper = mount(
-        <ContentLoader>
-          <Rect />
-          <Circle />
-        </ContentLoader>
+          <ContentLoader>
+            <rect x="80" y="17" rx="4" ry="4" width="300" height="13" />
+            <rect x="82" y="44" rx="3" ry="3" width="250" height="10" />
+            <circle cx="35" cy="35" r="35" />
+          </ContentLoader>
       )
 
       it('should render custom element', () => {
-        expect(wrapper).to.have.descendants(Rect)
-        expect(wrapper).to.have.descendants(Circle)
+          expect(wrapper.find('rect')).to.have.length(3)
+          expect(wrapper.find('circle')).to.have.length(1)
```

Comment:
The indentation of code is wrong here and above
But again, it isn't a big problem :)